tests/test_compat.py

```diff
@@ -51,7 +51,7 @@ def test_text_alternate_encoding(self):
         self.assertTrue(isinstance(result, bytes))
         self.assertEqual(result, b'La Pe\xc3\xb1a')
 
-class Test_cgi_FieldStorage_Py3_tests(object):
```

Comment:
The only reason the other tests in this file still use `unittest.TestCase` is due to the fact that they have not yet been converted to use `assert`.  See https://github.com/Pylons/webob/issues/245